modules/presets/field.js

```diff
@@ -10,6 +10,11 @@ export function presetField(fieldID, field, allFields) {
   allFields = allFields || {};
   let _this = Object.assign({}, field);   // shallow copy
 
+  // This handles fieldIDs that contain ':', like 'source:1'
+  let localizerFieldID = fieldID.includes(':') ? fieldID.split(':')[0] + '_multiple': fieldID;
+  // This is what comes after the ':' in the fieldID. 
+  let localizerOption = fieldID.includes(':') ? fieldID.split(':')[1]: 0;
```

Comment:
Nit: This would be a little easier to read if you use a normal if-else statement, maybe with a [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).